core/src/main/java/com/linecorp/armeria/client/HttpClientFactory.java

```diff
@@ -146,6 +149,8 @@
         useHttp1Pipelining = options.useHttp1Pipelining();
         connectionPoolListener = options.connectionPoolListener();
         meterRegistry = options.meterRegistry();
+        keepAliveTimer = MoreMeters.newTimer(meterRegistry, "armeria.client.connections.lifespan",
+                                             ImmutableList.of());
```

Comment:
Shouldn't we add some tags? e.g. protocol

Reply:
Oh, good idea!